.kuzzlerc.sample

```diff
@@ -212,6 +212,10 @@
     // The profileIds applied to a user created with the API action
     // "security:createRestrictedUser"
     "restrictedProfileIds": ["default"],
+    // [supportBrowserCookieAsJwt]
+    // Enable Kuzzle secure support of Jwt in Cookie for the browser
+    // This option however is not compatible with [http.accessControlAllowOrigin] set to '*'
+    "supportBrowserCookieAsJwt": false,
```

Comment:
Nitpicking: I don't really like that name, also because we try to avoid the term `jwt` and use authentication token instead.

Proposals:
  - `authenticateWithCookie`
  - `supportCookieAuthentication` 
  - ...